common/firmware/gpio.h

```diff
@@ -13,6 +13,8 @@ void gpio_reset(void* port, uint16_t pin, uint8_t active_setting);
 
 bool gpio_is_set(void* port, uint16_t pin, uint8_t active_setting);
 
+uint8_t gpio_read(void* port, uint16_t pin);
```

Comment:
Would this be for an analog input pin?

Reply:
oops, leftover test function,
nice catch